doc/vaulted-edit.1.md

```diff
@@ -27,8 +27,10 @@ AWS KEY
    Manages MFA (multi-factor authentication) ARN or serial number. If enabled, user will be prompted
    to enter MFA code when accessing vault. MFA will remain active for the
    vault duration.
+   If you are using STS credentials (see the 't' option below), you must have MFA enabled to invoke any IAM calls.
```

Comment:
Perhaps we should call out that the use of temporary credentials is the default?